src/app/cake/handlers/get.handler.ts

```diff
@@ -0,0 +1,10 @@
+import { Req, Res } from '@customTypes';
+import { RequestHandler } from 'express';
+import { getCakes } from '../cake.service';
+
+export const handler: RequestHandler[] = [
+  async (_: Req, res: Res) => {
```

Comment:
Use meaningful names for your variables, even are not used. `_` is meaningless and can cause confusion.

Reply:
Agree, I just didn't want my linter to warn me about and unused vars.

Reply:
Solved